### ISSUE TITLE
Scaling xgboost example does not show comparison of runtimes

### ISSUE THREAD

**Issue by therriault** [coiled/coiled-examples#6]
In https://github.com/coiled/coiled-examples/blob/master/scaling-xgboost/scaling-xgboost.ipynb, the `%%time` magic is used for the dask modeling step, but not for the sklearn modeling step, so you can't actually compare the results. (The Dask version shouldn't necessarily be faster because it uses a lot more data, but it'd help to understand the value to show both approaches' runtimes and amount of data used.)

Also, note that the AUC metric for both models is the same (0.73), which is a bit surprising given that the Dask version was trained on way more data. Maybe the default XGBoost hyperparameters aren't allowing the model to benefit from the additional data? Not sure, but this result does kind of undermine the argument for using Dask here, since you get the same result as you did when running locally. Maybe tuning the model or finding another dataset would make the example better?
